### ISSUE TITLE
How to add pull quotes programatically

### ISSUE THREAD

**Issue by kevinlisota** [alleyinteractive/apple-news#541]
Your plugin offers a meta box to enter or edit a pull quote. 

On our site, we have the meta box turned off for our writers. We also use a shortcode function to generate pull quote markup on our regular site.

Since we are inserting pull quotes programatically via a shortcode, we would be able to insert an Apple News pull quote programaically as well, without using the metabox.

I'm looking for pointers or documentation on how to achieve this.

**Comment by kevinfodness** [comment]
Yes, you can modify the resultant JSON in whatever way you like using the `apple_news_generate_json` filter: https://github.com/alleyinteractive/apple-news/wiki/Action-and-Filter-Hooks#includesapple-exporterclass-exporterphp
